.github/workflows/osrm-backend.yml

```diff
@@ -518,8 +518,14 @@ jobs:
         if [[ -n ${CLANG_VERSION} ]]; then
           echo "CCOMPILER=clang" >> $GITHUB_ENV
           echo "CXXCOMPILER=clang++" >> $GITHUB_ENV
-          ${MASON} install clang++ ${CLANG_VERSION}
-          echo "$(${MASON} prefix clang++ ${CLANG_VERSION})/bin" >> $GITHUB_PATH
+          sudo apt-get update -y && sudo apt-get install "clang++-${CLANG_VERSION}"
+          if [[ "${CLANG_VERSION}" = "6" ]]; then
```

Comment:
Clang 6 is installed as `/usr/bin/clang++-6.0` and Clang 11 as `/usr/bin/clang++-11`, so had to do this.
It is a bit ugly, but couldn't find better way.

Reply:
Is this the same package as https://packages.ubuntu.com/focal/clang-6.0?

Reply:
I think so.

Reply:
<img width="927" alt="Screenshot 2022-08-29 at 17 42 08" src="https://user-images.githubusercontent.com/266271/187240239-497263f3-c815-4e89-a6ed-e817ce992b7a.png">

Reply:
Is it possible to set CCOMPILER/CXXCOMPILER to e.g. clang-6.0/clang++6.0 and then just install all compilers at L543? Or is there a step that's requiring the symlink?

Reply:
Will try

Reply:
It seems it works, a way more elegant solution, thanks :+1: